docs/install-bitbucket-server.md

```diff
@@ -86,6 +86,11 @@
            // JFrog user and password with 'read' permissions for Xray
            // JF_USER= credentials("JF_USER")
            // JF_PASSWORD= credentials("JF_PASSWORD")
+   
+           // [Optional]
+           // Relevant for air-gapped environments.
+           // Name of the remote repository that Frogbot and its dependencies will be downloaded to.
```

Comment:
I suggest rephrasing as follows (applies to all installation files of course).

           // [Optional]
           // If the machine that runs Frogbot has no access to the internat, set the name of a remote repository 
           // in Artifactiry, which proxies https://releases.jfrog.io/artifactory
           // The 'frogbot' executable and other tools it needs will be downloaded through this repository.

Reply:
done

Reply:
done